README.md

```diff
@@ -22,6 +22,13 @@ Auditor.**
 Instead we recommend that you follow our guide on how to safely
 [collect credit information](https://docs.opn.ooo/collecting-card-information).
 
+## Compatibility
+
+OmiseGO version `v1.2.0` and up is designed to work with Go version 1.16 or higher. It is not compatible with Go versions 1.15 and below.
```

Comment:
I'd rather not use the name "OmiseGO"; instead I would say "version `v1.2.0` and up of this package..."

Reply:
Thanks. Updated.